package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@babel/standalone": "^7.22.9",
+    "@magicul/next-google-tag-manager": "^1.0.0",
```

Comment:
@niels-bosman this should probably be v2 already?

Reply:
@kgoedecke It is not released yet so I think we should be fine for now.